test/auditing/test_runner.py

```diff
@@ -243,3 +243,18 @@ def test_run_checks_and_save_results__failing_check(repository, fake_path, mocke
 )
 def test_determine_issue_status(is_found, old_status, new_status):
     assert uut.determine_issue_status(is_found, old_status) == new_status
+
+
+def test_notify_status_change(mocker, issue_factory, service_factory):
+    service = service_factory()
+    issue = issue_factory(repository=service.repository)
+    slack = mocker.Mock()
+    mocker.patch("zoo.auditing.runner.slack", slack)
+    uut.notify_status_change(issue)
+    audit_url = f"services/{issue.repository.owner}/{issue.repository.name}/auditing/"
+    text = "{status} issue {issue} on <{repo.url}|{repo.name}>.".format(
```

Comment:
Hmm, could this be a constant? So we don't have to update the test if we just change the string... wdyt?

Reply:
Sorry I don’t get it actually: even in `text` is static, if the message is change in the runner function, this text has to change. What am I missing?